src/DotNetProjectFile.Analyzers/MsBuild/OutputType.cs

```diff
@@ -1,5 +1,17 @@
 namespace DotNetProjectFile.MsBuild;
 
+/// <summary>
+/// Specifies the file format of the output file. This parameter can have one
+/// of the following values:
+/// - Library: Creates a code library. (Default value.)
+/// - Exe:    Creates a console application.
+/// - Module: Creates a module.
+/// - Winexe: Creates a Windows-based program.
```

Comment:
Isn't this supposed to be `WinExe` rather than `Winexe`?

Reply:
Those values are not case-sensitive, and just copied from MS. I'll check the casing though.